packages/web-helpers/src/WebUser.ts

```diff
@@ -185,43 +238,45 @@ export class WebUser extends EventEmitter {
         return this.connecting
     }
 
-    public async disconnect(): Promise<void> {
+    /**
+     * Disconnect the user from the Verida Network.
+     *
+     * @emit disconnected When the user is successfully logged out.
+     */
+    public async disconnect() {
         try {
-            const context = await this.getContext()
+            const context = this.getContext()
             await context.disconnect()
 
             this.context = undefined
             this.account = undefined
-            this.profile = undefined
             this.did = undefined
+            this.profile = undefined
+            this.profileConnection = undefined
             this.connecting = undefined
 
             if (this.config.debug) {
                 console.log(`Account disconnected`)
             }
 
             this.emit('disconnected')
-        } catch (err: any) {
-            if (err.message.match('Not connected')) {
+        } catch (error: any) {
+            if (error.message.match('Not connected')) {
                 return
             }
-
-            throw err 
+            throw error
         }
     }
 
     /**
-     * Send a generic message to a user's Verida Wallet
-     * 
-     * @param {*} did 
-     * @param {*} subject 
-     * @param {*} message 
-     * @param {*} linkUrl 
-     * @param {*} linkText 
+     * Send a generic message to a user's inbox (accessible from the Verida Wallet).
+     *
+     * @param {string} did
+     * @param {WebUserMessage} message the message definition
      */
-    public async sendMessage(did: string, message: WebUserMessage): Promise<void> {
-        const context = await this.getContext()
-        const messaging = await context!.getMessaging()
+    public async sendMessage(did: string, message: WebUserMessage) {
```

Comment:
Must be async

Reply:
It is async, we just don't need the syntax as it's inferred by the return of the Promise from `messaging.send`.